src/main.py

```diff
@@ -1,12 +1,18 @@
-
 from vex import *
 
+#CONFIG
 
 brain=Brain()
 
 controller=Controller(PRIMARY)
 
 
+
+

```

Comment:
Can you reduce the blank lines throughout the repo? At most should be 2-3 blank lines, no more as harder to navigate